go.mod

```diff
@@ -138,7 +138,7 @@ require (
 	github.com/gorilla/mux v1.8.0 // indirect
 	github.com/gorilla/websocket v1.5.0 // indirect
 	github.com/h2non/filetype v1.1.1 // indirect
-	github.com/hashicorp/cronexpr v1.1.0 // indirect
+	github.com/hashicorp/cronexpr v1.1.2 // indirect
```

Comment:
Q: will the fix work without dep bump?

Reply:
It is auto-detected as Apache-2.0 although the file is not called LICENSE. Is the file name the problem?